docs/ref/models/expressions.txt

```diff
@@ -373,6 +373,15 @@ should define the desired ``output_field``. For example, adding an
 The ``**extra`` kwargs are ``key=value`` pairs that can be interpolated
 into the ``template`` attribute.
 
+The ``filter`` argument requires a ``django.db.Q`` object which will be used
+to filter the rows that are aggregated. See the
+:doc:`Aggregation guide </topics/db/aggregation>` for more information on
```

Comment:
_Filtering on annotations_ 

You should probably make a link for that section.